docs/src/main/sphinx/release/release-354.md

```diff
@@ -0,0 +1,56 @@
+# Release 354 (XXXXXXXX Mar 2021)
+
+## General
+
+* Preferred write partitioning is now triggered automatically when the number
```

Comment:
how about briefly mentioning how this default setting helps? might be good for people that haven't followed previous release notes.

Reply:
cc @sopel39

Reply:
s/triggered/selected?

Reply:
> how about briefly mentioning how this default setting helps? might be good for people that haven't followed previous release notes.

This note is already long. Maybe we could add at the end:
```
Preferred write partitioning overcomes maximum number
of partitions per writer limit in Hive connector.
```
?

Reply:
This seems to be missing the word "estimated"

> when the estimated number of written partitions will exceed